### PR TITLE
Update to node 10 in .travis.yml

### PR DESCRIPTION
**Pull request by tripu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,9 +45,9 @@
   "dependencies": {
     "body-parser": "1.18.2",
     "compression": "1.7.2",
-    "express": "5.0.0-alpha.6",
+    "express": "4.16.3",
```
Comment (deniak) [inline package.json]: Why using an older version?

File: package.json [hunk 2]
```diff
@@ -45,9 +45,9 @@
...
     "file-type": "7.7.1",
-    "immutable": "4.0.0-rc.4",
+    "immutable": "3.8.2",
```
Comment (deniak) [inline package.json]: Why using an older version?

File: package.json (hunk 1 above)
Comment (tripu) [reply package.json]: https://github.com/w3c/specberus/pull/752#discussion_r189235375

File: package.json (hunk 2 above)
Comment (tripu) [reply package.json]: https://github.com/w3c/specberus/pull/752#discussion_r189235375

**Pull request merged**